packages/front-end/services/UserContext.tsx

```diff
@@ -192,7 +192,9 @@ export function UserContextProvider({ children }: { children: ReactNode }) {
     data: currentOrg,
     mutate: refreshOrganization,
     error: orgLoadingError,
-  } = useApi<OrgSettingsResponse>(isAuthenticated ? `/organization` : null);
+  } = useApi<OrgSettingsResponse>(
+    isAuthenticated && data?.userId ? `/organization` : null
```

Comment:
Does this affect other places too? Should we change isAuthenticated to only be set to true once we have a user ID instead?